db/crud.go

```diff
@@ -1094,6 +1093,14 @@ func (db *Database) resolveConflict(localDoc *Document, remoteDoc *Document, doc
 		return "", nil, resolveFuncError
 	}
 
+	// Restore rev ID onto localDoc and remoteDoc in the event that these have been mutated inside a custom conflict
+	// resolver
+	localDoc.RevID = localRevID
```

Comment:
The versions of localDoc and remoteDoc we're passing to the conflict resolver (inside Conflict) should already be distinct copies - I didn't think this reassignment would be necessary?